### PR TITLE
docs(cdk): update usage notes of coalesceWith-operator

### PR DESCRIPTION
**Pull request by mikelgo**
* wrong import from 'ngrx/component'
* wrong function-name
* missing durationSelector

### CODE REVIEW

File: libs/cdk/coalescing/src/lib/coalesceWith.ts [hunk 1]
```diff
@@ -31,11 +31,12 @@ import { coalescingManager } from './coalescingManager';
  * @usageNotes
  * Emit clicks at a rate of at most one click per second
  * ```typescript
- * import { fromEvent, animationFrames } from 'rxjs';
- * import { coalesce } from 'ngRx/component';
+ * import { interval, fromEvent, animationFrames } from 'rxjs';
```
Comment (edbzn) [inline libs/cdk/coalescing/src/lib/coalesceWith.ts:34]:
```suggestion
 * import { interval, fromEvent } from 'rxjs';
```

File: libs/cdk/coalescing/src/lib/coalesceWith.ts [hunk 2]
```diff
@@ -31,11 +31,12 @@ import { coalescingManager } from './coalescingManager';
...
+ * import { coalesceWith } from '@rx-angular/cdk/coalescing';
  *
+ * const setTimeoutDurationSelector = interval(500) as Observable<any>;
```
Comment (edbzn) [inline libs/cdk/coalescing/src/lib/coalesceWith.ts:37]:
```suggestion
 * const setTimeoutDurationSelector = interval(500);
```

**Review by edbzn — COMMENTED** [review]
Added 2 minor comments.

**Comment by mikelgo** [comment]
> Added 2 minor comments.

Thanks for also taking care of the polishing =)

**Pull request merged**